popup/popup.js

```diff
@@ -135,7 +170,11 @@ showStats = () => {
   while (equivalenceTitle.firstChild) {
     equivalenceTitle.removeChild(equivalenceTitle.firstChild);
   }
-  equivalenceTitle.appendChild(document.createTextNode(browser.i18n.getMessage('equivalenceTitle', [duration.toString(), megaByteTotal, kWhTotal.toString(), gCO2Total.toString()])));
+
+  const text = document.createElement("div");
+  text.innerHTML = browser.i18n.getMessage('equivalenceTitle', [duration.toString(), megaByteTotal, kWhTotal.toString(), gCO2Total.toString()]);
```

Comment:
`.innerHTML` is not recommended by Firefox Addons platform. It is why nodes are created manually. We will try to publish this version as it.